engine/transcoder/ffmpeg.go

```diff
@@ -38,13 +38,14 @@ func (ff *ffmpeg) Start(ctx context.Context, command, path string, maxBitRate in
 }
 
 // Path will always be an absolute path
-func createTranscodeCommand(cmd, path string, maxBitRate int, format string) (string, []string) {
+func createTranscodeCommand(cmd, path string, maxBitRate int, format string) []string {
 	split := strings.Split(cmd, " ")
 	for i, s := range split {
 		s = strings.Replace(s, "%s", path, -1)
 		s = strings.Replace(s, "%b", strconv.Itoa(maxBitRate), -1)
+		s = strings.Replace(s, "%f", format, -1)
```

Comment:
Instead of handling `%f` in the command, we should actually remove the `format` parameter. This parameter is a left over from a previous (flawed) transcoding implementation, that I forgot to remove. It does not make sense to be parameterizable as it will aways be the same, for a given transcoding setting.